ARM_VERSION

```diff
@@ -1 +1 @@
-0.11.0
\ No newline at end of file
+0.11.0
```

Comment:
Do we have a plan to move the things from `master` to `azure-stack` branch since last released yesterday? If not, we should list down what should be brought from `master` to `azure-stack` before merging of after merging this one.

Reply:
We are going to move things from master to azure-stack. So, No action here

Reply:
Perfect. Let's make a list of things we'll be bringing